### PR TITLE
[BPK-848]update deployment target

### PR DESCRIPTION
**Pull request by matteo-hertel**

### CODE REVIEW

**Comment by matteo-hertel** [comment]
yes indeed!

**Comment by shaundon** [comment]
Well that's convenient! ✋ :five:

**Comment by matteo-hertel** [comment]
This is all done and tested but we'll wait for the new domain before merging it

File: packages/bpk-docs/src/Readme.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Backpack Design System Documentation
```
Comment (matthewdavidson) [inline packages/bpk-docs/src/Readme.md]: This should be `README.md`.

File: webpack.config.babel.js [hunk 2]
```diff
@@ -203,6 +204,10 @@ if (isProduction) {
         warnings: false,
       },
     }),
+    new CopyWebpackPlugin([
+      { from: 'packages/bpk-docs/src/Readme.md', to: 'Readme.md' },
```
Comment (matthewdavidson) [inline webpack.config.babel.js]: Does this get copied to the `dist` directory?

Comment (matteo-hertel) [reply webpack.config.babel.js]: yes it does!

File: webpack.config.babel.js [hunk 3]
```diff
@@ -203,6 +204,10 @@ if (isProduction) {
         warnings: false,
       },
     }),
+    new CopyWebpackPlugin([
+      { from: 'packages/bpk-docs/src/README.md', to: 'README.md' },
```
Comment (k0nserv) [inline webpack.config.babel.js]: Couldn't we just create the `README.md` in https://github.com/backpack/backpack.github.io once and then leave it?

Comment (matteo-hertel) [reply webpack.config.babel.js]: Travis will force push to the target branch so it'll be nuked

Comment (k0nserv) [reply webpack.config.babel.js]: Really? So it will always just have a single commit?

Comment (matteo-hertel) [reply webpack.config.babel.js]: according to the docs yes
https://docs.travis-ci.com/user/deployment/pages/

Comment (k0nserv) [reply webpack.config.babel.js]: Cool

**Pull request merged**